### PR TITLE
Updating Copyright information

### PR DESCRIPTION
**Pull request by perpetualKid**

### CODE REVIEW

File: Source/Contrib/ActivityEditor/ActivityEditor/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -9,8 +9,8 @@
 [assembly: AssemblyDescription("Open Rails Transport Simulator")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Open Rails")]
-[assembly: AssemblyProduct("Open Rails (disabled)")]
```
Comment (twpol) [inline Source/Contrib/ActivityEditor/ActivityEditor/Properties/AssemblyInfo.cs]: We actually should leave this alone, as this is how this project is hidden from the main menu's dropdown (which is due to it not working properly IIRC).

**Review by twpol — CHANGES_REQUESTED** [review]
Just put the product name back for the activity editor and we're good here 🙂

File: Source/Contrib/ActivityEditor/ActivityEditor/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (perpetualKid) [reply Source/Contrib/ActivityEditor/ActivityEditor/Properties/AssemblyInfo.cs]: not the most obvious way...

Comment (twpol) [reply Source/Contrib/ActivityEditor/ActivityEditor/Properties/AssemblyInfo.cs]: You are not wrong there! It actually works by comparing the executable's product name with that of the main menu, and if it is different it is excluded. This was originally to make sure oddities like Visual Studio's "foo.vshost.exe" didn't appear but was co-opted here. Obviously the real solution would be to fix this contributed app or remove it. 🙂

**Review by twpol — APPROVED** [review]
Thanks for all the housekeeping you're doing! 🎉

**Pull request merged**